lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
I agree it is backwards incompatible and would need a minor version bumb, but I think it is a good change. 

* what a new user would expect
* paves the road for doing things like schema loading and dumping, without additional confusion

For reference, 0.2.4 is backwards incompatible to 0.2.3 in the same way as this proposal (renamed `setup` to `reset`). This is proposing making that name change just different to something a bit more correct in my mind.